handbook/digital-experience/README.md

```diff
@@ -319,6 +319,8 @@ Time management for the CEO is essential.  The Apprentice processes the CEO's ca
        - LinkedIn messages (careful not to mark things as read!)
        - Google Drive 
   - Be sure to do this from the CEO's browser so as to not lock him out of any meeting docs.
+  - Change “Notes” into “Agenda” when the document is ready for the CEO to review prior to the meeting.
+  - Connect the CEO with the person they are meeting with on LinkedIn (Do not send a note when doing this).
```

Comment:
```suggestion
```
I would add this above, by this time you already had to find this person on LinkedIn. It doesn't make sense to send the connect request at the end.